### ISSUE TITLE
resize key mappings don't work

### ISSUE THREAD

**Issue by mvolkmann** [AstroNvim/AstroNvim#1742]
### Checklist

- [X] Using a stable version of Neovim (i.e. not neovim nightly)
- [X] `:AstroUpdate`
- [X] Restarted AstroNvim

### Operating system/version

macOS 13.2.1

### Terminal/GUI

tried in both Terminal and Warp

### AstroNvim Health

 astronvim: require("astronvim.health").check()
    2 ========================================================================
    3 ## AstroNvim
    4   - INFO: AstroNvim Version: v3.8.0
    5   - INFO: Neovim Version: v0.8.3
    6   - OK: Using stable Neovim >= 0.8.0
    7   - OK: `git` is installed: Used for core functionality such as updater and plugin management
    8   - OK: `open` is installed: Used for `gx` mapping for opening files with system opener (Optional)
    9   - OK: `lazygit` is installed: Used for mappings to pull up git TUI (Optional)
   10   - OK: `node` is installed: Used for mappings to pull up node REPL (Optional)
   11   - OK: `gdu` is installed: Used for mappings to pull up disk usage analyzer (Optional)
   12   - OK: `btm` is installed: Used for mappings to pull up system monitor (Optional)
   13   - OK: `python3` is installed: Used for mappings to pull up python REPL (Optional)

### Describe the bug

When I have multiple splits open and press ctrl and any arrow key nothing happens.
I know that is supposed to resize the splits, but it doesn't work for me.

### Steps to Reproduce

1. Open multiple splits.
2. Press ctrl-any arrow key.
3. Note that nothing happens.

### Expected behavior

Splits should resize.

**Comment by mehalter** [comment]
@mvolkmann if you do `:Telescope keymap` can you search and see the mappings getting set or being set to something else?

**Comment by mehalter** [comment]
With a fresh installation I cannot replicate this, and I can see these mappings being set correctly, if you cannot see them please let me know and I can reopen this and investigate. If you do see these mappings and it doesn't work, then this would be a bug with `smart-splits` and not AstroNvim. 

![2023-04-06_07:40:54_screenshot](https://user-images.githubusercontent.com/1591837/230366526-47c833e4-2443-4040-b6f1-81c4ad537567.png)

**Comment by mvolkmann** [comment]
I found the solution. It was not related to anything in my Neovim configuration and not related to the "smart-splits" plugin. The issue was default macOS key bindings. Surely I'm not the first Mac user to run into this. The attached screenshot shows the key mappings I had to turn off in System Settings. Now I can resize editing panes just fine using ctrl and the arrow keys.

<img width="678" alt="smart-splits-keys" src="https://user-images.githubusercontent.com/79312/230691365-b7f1ec5d-81ba-4ae9-8848-94e22dd18d5a.png">
